src/backend/Order/Actions/StocksActions/StocksGetList/StocksGetListItemResponse.cs

```diff
@@ -6,5 +6,6 @@ public class StocksGetListItemResponse
         public string ProductName { get; set; } = default!;
         public int Quantity { get; set; } = default!;
         public string Unit { get; set; } = default!;
+        public string WareHouseId { get; set; } = default!;
```

Comment:
`WarehouseId`, to co napisałeś zostałoby prztłumaczone na json `wareHouseId`

Reply:
fixed